modules/UI/videolayout/Filmstrip.js

```diff
@@ -39,30 +39,51 @@ const Filmstrip = {
         const thumbs = this._getThumbs(!forceUpdate);
         const avatarSize = height / 2;
 
-        if (thumbs.localThumb) {
-            thumbs.localThumb.css({
-                'padding-top': '',
-                height: `${height}px`,
-                'min-height': `${height}px`,
-                'min-width': `${width}px`,
-                width: `${width}px`
-            });
-        }
+        const bigVideoCSS = {
+            'padding-top': '',
+            height: `${height}px`,
+            'min-height': `${height}px`,
+            'min-width': `${width}px`,
+            width: `${width}px`
+        };
 
-        if (thumbs.remoteThumbs) {
-            thumbs.remoteThumbs.css({
-                'padding-top': '',
-                height: `${height}px`,
-                'min-height': `${height}px`,
-                'min-width': `${width}px`,
-                width: `${width}px`
-            });
+        if (thumbs.localThumb) {
+            thumbs.localThumb.css(bigVideoCSS);
         }
 
         $('.avatar-container').css({
             height: `${avatarSize}px`,
             width: `${avatarSize}px`
         });
+
+        [...thumbs.remoteThumbs].forEach(videoThumb => {
+            const $thumb = $(videoThumb);
+
+            // Smaller video
+            if ($thumb.hasClass('without-camera')) {
```

Comment:
please don't use any `else`'s, rather use early-returns. It's very important rule (mandatory).
To do that generally put "unhappy paths" to be first (this is checking for errors, not-logged in users etc), in this case there are none. Then follow with "happy paths', where the "default-behavior" usually comes first. The structure should consist of several linear `if (case) { doStuff; return; }`. Also keeping the function body in the 50-lines constraint is the best.
```
// this is a happy-path but handles a default-behavior
if (!$thumb.hasClass..) {
    $thumb.css(bigVideoCSS);
    return;
}
// this is happy-path but rather a main logic (the biggest block)
const ratio = width / height;
...
```

Reply:
fixed